cterasdk/core/plans.py

```diff
@@ -213,12 +217,12 @@ def set_policy(self, rules, apply_default=None, default=None, apply_changes=True
         if apply_default is False:
             policy.defaultPlan = None
         elif apply_default is True and default:
-            policy.defaultPlan = self._portal.plans.get(default, ['baseObjectRef']).baseObjectRef
+            policy.defaultPlan = portal_plans.get(default).baseObjectRef
 
         policy_rules = []
         for rule in rules:
-            policy_rule = PolicyRuleCoverter.convert(rule, 'PlanAutoAssignmentRule', 'plan')
-            policy_rule.plan = portal_plans.get(rule.value).baseObjectRef
+            policy_rule = PolicyRuleCoverter.convert(rule, 'PlanAutoAssignmentRule', 'plan',
```

Comment:
Since its a single line with append, you can change it to list comprehension:
policy_rules = [PolicyRuleCoverter.convert(...) for rule in rules]